src/primitives/transaction.h

```diff
@@ -399,8 +399,8 @@ template <typename Tx> static inline CTransactionRef MakeTransactionRef(Tx&& txI
 /** A generic txid reference (txid or wtxid). */
 class GenTxid
 {
-    const bool m_is_wtxid;
-    const uint256 m_hash;
+    bool m_is_wtxid;
+    uint256 m_hash;
```

Comment:
in commit ff290ca931f7131b3c1f7b7ac62b7a58ecc3c794:

Any reason for this change? gcc and clang (c++17) compile fine for me without it.

If you want to modify this class in this pull, you might add a `ToString()const` method that returns `IsWtxid() ? "wtx " : "tx " + GetHash().ToString()` and use it where appropriate.

Reply:
The std::sort (in unit tests) on pairs of NodeId, GenTxid failed without GenTxid having an assignment operator for me.

Reply:
Thanks. Didn't compile the fuzz tests.